src/sentry/models/sentryfunction.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+from sentry.db.models import BaseManager, DefaultFieldsModel, FlexibleForeignKey
+
+
+# TODO: remove
+class SentryFunctionManager(BaseManager):
+    def get_alertable_sentry_functions(self, organization_id):
+        # TODO: check events
+        functions = self.filter(
+            organization_id=organization_id,
+        )
+        return filter(lambda fn: "alert" in fn.organization, list(functions))
+        # needs to be changed
+
+
+class SentryFunction(DefaultFieldsModel):
+    __include_in_export__ = False
+
+    organization = FlexibleForeignKey("sentry.Organization")
+    name = models.TextField()
+    slug = models.CharField(max_length=64, unique=True)
+    author = models.TextField(null=True)
```

Comment:
Is author optional? Otherwise, you can drop null here since you're creating a new table.